services/dkp-insights-management/0.10.0/dkp-insights-management.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: dkp-insights-management
+  namespace: ${releaseNamespace}
+spec:
+  chart:
+    spec:
+      chart: dkp-insights-management
+      sourceRef:
+        kind: HelmRepository
+        name: mesosphere.github.io-dkp-insights-charts-management
+        namespace: kommander-flux
+      version: v0.10.0-rc1
```

Comment:
the manifest validation step is complaining about this chart not existing in the repo. Is this mean to be `v0.1.0-rc1` instead or is this version just not released yet?

```
  2022-03-25 06:06:52 ERR HelmRelease dkp-insights-management err="chart "dkp-insights-management" not found in https://mesosphere.github.io/dkp-insights-charts-management repository"
```

Reply:
@cbuto so I issued the following:
```
helm repo add dkp-insights-management  https://mesosphere.github.io/dkp-insights-charts-management/
helm repo update
````
This yielded no results:
```
helm search repo dkp-insights-management
No results found
```
But this did:
```
helm search repo dkp-insights-management --devel                                                                                                       
NAME                                            CHART VERSION   APP VERSION     DESCRIPTION
dkp-insights-management/dkp-insights-management v0.1.0-rc1  
```

So I suspect it doesn't like the `-rc` in the version name and can't retrieve any versions that you explicitly need the `--devel` flag.

Reply:
Oh, this was a terrible typo. Apologies.